README.md

```diff
@@ -155,33 +159,45 @@ Update the project:
 ```bash
 npx cap update ios
 ```
+Sync the project:
 
+```bash
+npx cap sync ios
+```
 Make the following changes to your `AppDelegate.swift` file:
+> To consider: Read the official documentation. The import method is different depending on the swift version. Capacitor version 1.4.0 uses Swift version 4.2. 
+[Check version](https://github.com/ionic-team/capacitor/blob/master/.swift-version)
 
 ```diff
 + import Branch
 
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
     // Override point for customization after application launch.
 +   Branch.setUseTestBranchKey(true) // if you are using the TEST key
-+   Branch.getInstance()!.initSession(launchOptions: launchOptions)
++   Branch.getInstance().initSession(launchOptions: launchOptions)
     return true
   }
 
   func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
     // Called when the app was launched with a url. Feel free to add additional processing here,
     // but if you want the App API to support tracking app url opens, make sure to keep this call
-+   Branch.getInstance()!.application(app, open: url, options: options)
++   Branch.getInstance().application(app, open: url, options: options)
     return CAPBridge.handleOpenUrl(url, options)
   }
 
   func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
     // Called when the app was launched with an activity, including Universal Links.
     // Feel free to add additional processing here, but if you want the App API to support
     // tracking app url opens, make sure to keep this call
-+   Branch.getInstance()!.continue(userActivity)
++   Branch.getInstance().continue(userActivity)
     return CAPBridge.handleContinueActivity(userActivity, restorationHandler)
   }
+
+// Optional (You probably have to add the entire function)
+  func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+  // handler for Push Notifications
++  Branch.getInstance().handlePushNotification(userInfo)
+}
```

Comment:
Because this entire function doesn't exist in the Capacitor iOS template, can you add `+ ` before each line?